test/connectors/v3/getFinancialDetails/GetFinancialDetailsConnectorSpec.scala

```diff
@@ -27,20 +27,26 @@ import org.mockito.Mockito._
 import play.api.http.Status
 import play.api.test.Helpers.{await, defaultAwaitTimeout}
 import uk.gov.hmrc.http.{HeaderCarrier, HttpClient}
+import utils.DateHelper
 
 import scala.concurrent.{ExecutionContext, Future}
 
-class FinancialDetailsConnectorSpec extends SpecBase {
+class GetFinancialDetailsConnectorSpec extends SpecBase {
   implicit val ec: ExecutionContext = ExecutionContext.Implicits.global
   val mockHttpClient: HttpClient = mock(classOf[HttpClient])
   val mockAppConfig: AppConfig = mock(classOf[AppConfig])
+  val mockDateHelper: DateHelper = mock(classOf[DateHelper])
+  val sampleDate: LocalDate = LocalDate.of(2023, 1, 1)
 
   class Setup {
     reset(mockHttpClient)
     reset(mockAppConfig)
```

Comment:
Could you add a reset(mockDateHelper) here please
```suggestion
    reset(mockAppConfig)
    reset(mockDateHelper)
```